configuration-as-code-examples/cloudbees-assurance-plugin/casc_disabled.yaml

```diff
@@ -0,0 +1,10 @@
+# This file shows how it is possible to disable the CloudBees Assurance Program using Configuration as Code.
+# CloudBees does not recommend you to disable it, however.
```

Comment:
```suggestion
# CloudBees does not recommend that to disable it, however.
```

Reply:
@KimberlyFields you maybe mean "that you"?

Reply:
Yes, it should be "CloudBees does not recommend that you disable it, however."

Reply:
Changed!